[loops.assertion.1] line 32 assertion : FAILURE
VERIFICATION FAILED
